src/api/image/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
-    "@senecacdot/satellite": "^1.3.0",
+    "@senecacdot/satellite": "^1.6.0",
```

Comment:
Given the rapid development of Satellite, should we remove the version pinning until it's stable / standardized and then pin?

Reply:
Good point, I guess using `^1.6.0` will mean any `1.x` gets used already.  I was being explicit, but it will still pick-up the new versions.  I'm not planning to do a major release update on satellite this term I don't think.

Reply:
Updated to `^1.x`